docs/pages/server-access/getting-started.mdx

```diff
@@ -343,4 +343,4 @@ Feel free to shut down, clean up, and delete your resources or use them in furth
 - [Announcing Teleport SSH Server](https://goteleport.com/blog/announcing-teleport-ssh-server/)  
 - [How to SSH properly](https://goteleport.com/blog/how-to-ssh-properly/)  
 - Consider whether [OpenSSH or Teleport SSH](https://goteleport.com/blog/openssh-vs-teleport/) is right for you.  
-- Learn about [Teleconsole](https://goteleport.com/blog/instant-ssh-github/) built on Teleport 
\ No newline at end of file
+- Learn about [Teleconsole](https://goteleport.com/blog/instant-ssh-github/) built on Teleport 
```

Comment:
```suggestion
```

I know this is unrelated but it caught my eye. We've shutdown Teleconsole https://www.teleconsole.com/